.env.example

```
@@ -21,6 +21,13 @@ MONGODB_URI="mongodb://localhost:27017/mabar"
 # =============================================================================
 GEMINI_API_KEY="your_gemini_api_key_here"
 
+# =============================================================================
+# BACK4APP CONFIGURATION
+# =============================================================================
+VITE_BACK4APP_APP_ID="your_back4app_app_id_here"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)